packages/app/src/cli/services/app/config/push.ts

```diff
@@ -43,3 +47,13 @@ export async function pushConfig(options: Options) {
 export const abort = (errorMessage: OutputMessage) => {
   throw new AbortError(errorMessage)
 }
+
+// this is placeholder for a more robust validation/clearing layer
+export const removeFalsyEntries = (obj: {[key: string]: string | string[] | undefined}) => {
```

Comment:
we paired on this -- we'll want to remove this in favor of a better validation/deletion approach, but this will have to wait until our config schema and UX for deleting fields are more defined.

Reply:
Definitely worth revisiting but we identified in [this issue](https://github.com/Shopify/partners/issues/48089) the scenarios for what happens when each field is cleared from the file.